i18n/en/docusaurus-plugin-content-docs/version-5.0/12-observability/01metrics.md

```diff
@@ -54,45 +55,82 @@ The following table describes the labels of the metrics that are related to the
 
-
 ### Consumer metrics
+
 The following table describes the labels of the metrics that are related to the consumers in Message Queue for Apache RocketMQ.
 
 - topic: the topic of Message Queue for Apache RocketMQ.
 - consumer_group: the ID of the consumer group.
 - client_id: the ID of the client.
 - invocation_status: the result of the API call for sending messages, which includes success and failure.
 
-| Type | Name | Unit | Description | Label |
-| --- | ----- | --- | ---------- | --- |
-| Histogram | rocketmq_process_time | millisecond | The distribution of message process time.The following shows the distribution ranges: <br />le_1_ms <br />le_5_ms  <br />le_10_ms<br />le_100_ms <br />le_10000_ms<br />le_60000_ms <br />le_overflow | topic,consumer_group,client_id,invocation_status |
-| gauge | rocketmq_consumer_cached_messages | message | The number of messages in the local buffer queue of PushConsumer. | topic,consumer_group,client_id |
-| gauge | rocketmq_consumer_cached_bytes | byte | The total size of messages in the local buffer queue of PushConsumer. | topic,consumer_group,client_id |
-| Histogram | rocketmq_await_time | millisecond | The distribution of queuing time for messages in the local buffer queue of PushConsumer. The following shows the distribution ranges:<br />le_1_ms<br /> le_5_ms<br />le_20_ms<br />le_100_ms <br />le_1000_ms<br /> le_5000_ms <br />le_10000_ms<br />le_overflow | topic,consumer_group,client_id |
-
+| Type      | Name                              | Unit        | Description                                                                                                                                                                                                                                                        | Label                                            |
+| --------- | --------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------ |
+| Histogram | rocketmq_process_time             | millisecond | The distribution of message process time.The following shows the distribution ranges: <br />le_1_ms <br />le_5_ms  <br />le_10_ms<br />le_100_ms <br />le_10000_ms<br />le_60000_ms <br />le_overflow                                                              | topic,consumer_group,client_id,invocation_status |
+| gauge     | rocketmq_consumer_cached_messages | message     | The number of messages in the local buffer queue of PushConsumer.                                                                                                                                                                                                  | topic,consumer_group,client_id                   |
```

Comment:
Should the units here be aligned with the `rocketmq_messages_in_total` above? or which do you prefer? count or message.

Reply:
`rocketmq_consumer_cached_messages` is a gauge and `rocketmq_messages_in_total` is a counter. It makes sense that these two metrics with the different suffixes.